felix/dataplane/linux/endpoint_mgr.go

```diff
@@ -1436,8 +1444,15 @@ func (m *endpointManager) configureInterface(name string) error {
 		return nil
 	}
 
+	// When running as non-root, the host's '/proc' is mounted at '/nodeproc'
```

Comment:
Is there a reason to have this duplicated block from above?

Reply:
This is in a different func, but seeing as this is already showing up in 3 different places, it might warrant a helper func to dedupe... Thanks for catching it!